README.md

```diff
@@ -1,2 +1,196 @@
-# helm-operator
-Helm Operator for OpenEBS Installation
+# OpenEBS Helm Operator
+
+## Introduction
+
+- This is  a helm operator for openebs
+- The content of values.yaml will be placed as spec items in `OpenEBSInstallTemplate` CR 
+- The operator watches all namespaces. There is no need for helm client/tillerto be installed.
+  (the helm operator imports code from helm project)
+- Changes to spec of the CR will behave similar to values override via helm
+- All OpenEBS resources have ownerReference of `OpenEBSInstallTemplate` & are deleted upon removal of CR
+
+Example of OpenEBSInstallTemplate: 
```

Comment:
Should we push entire yaml as an example in README?
Does this mean, we should change this for every change to code?

Reply:
Good point @AmitKumarDas ..will refer to links instead of pasting the complete yaml!

Reply:
Done